specification/electronic-prescription-service-api.yaml

```diff
@@ -203,10 +203,13 @@ paths:
         '200':
```

Comment:
do 200 responses from other endpoints also return operation outcomes?

Reply:
No, /Convert returns xml, and /Prepare returns a Parameter resource.